app/addons/components/react-components.react.jsx

```diff
@@ -258,6 +258,23 @@ function (app, FauxtonAPI, React, Components, beautifyHelper) {
     }
   });
 
+  var LoadLines = React.createClass({
+
+    render: function () {
+
+      return (
+        <div id="loading-lines">
```

Comment:
can we switch it to a class? it gets more reusable, then we can have multiple areas on the page using the loadinglines.